.circleci/config.yml

```diff
@@ -260,6 +260,8 @@ jobs:
       - run:
           name: Tests TypeScript definitions
           command: pnpm typescript:ci
+          environment:
+            NODE_OPTIONS: --max-old-space-size=3072
```

Comment:
Had to increase memory to avoid https://app.circleci.com/pipelines/github/mui/material-ui/121186/workflows/715975a6-6753-464e-ba7e-38d860316544/jobs/652147/steps